woocommerce/woocommerce-android
# Issue/2452 main activity leaked

Closes #2452 by fixing a [leaky handler](https://www.androiddesignpatterns.com/2013/01/inner-class-handler-memory-leak.html) in `DashboardStatsView`. 

To test, first repo the problem in `develop`:

- Make sure Leak Canary notifications are enabled
- Make sure the beta stats are **not** enabled
- Go to settings
- Switch sites
- Return to the main screen

A second or two later you'll see the Leak Canary icon appear.

![Screenshot_1589914420](https://user-images.githubusercontent.com/3903757/82366961-0d6a8f00-99e1-11ea-9504-3508edbf150b.png)

If you view the leak report, you'll see that the main activity and primary fragments have been leaked.

Now try the same thing with this branch and verify that no leak is reported.

Update release notes:
